features/verblijfplaats.feature

```diff
@@ -0,0 +1,98 @@
+      | land (13.10) | 0000   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam                   | waarde                 |
+      | type                   | VerblijfplaatsOnbekend |
+      | verblijfplaatsOnbekend | true                   |
+
+  Rule: Een verblijfplaats is een adres of locatie in het buitenland wanneer land (13.10) een waarde ongelijk aan '0000' (onbekend) heeft
+    - veld 'type' krijgt de waarde 'VerblijfplaatsBuitenland'
+
+    Abstract Scenario: verblijfplaats buitenland
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam         | waarde |
+      | land (13.10) | <land> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam | waarde                   |
+      | type | VerblijfplaatsBuitenland |
+
+      Voorbeelden:
+      | land | land omschrijving            |
+      | 6014 | Verenigde Staten van Amerika |
+      | 9999 | Internationaal gebied        |
+
+  Rule: Een verblijfplaats is een binnenlands adres wanneer straat (11.10) en/of huisnummer (11.20) een waarde heeft
+    - veld 'type' krijgt de waarde 'Adres' 
+
+    Abstract Scenario: verblijfplaats binnenlands adres
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam               | waarde       |
+      | land (13.10)       |              |
+      | straatnaam (11.10) | <straat>     |
+      | huisnummer (11.20) | <huisnummer> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam | waarde |
+      | type | Adres  |
+
```

Comment:
Met deze voorbeelde worden er toch ook straat en/of huisnummer geretourneerd in de verblijfplaats. Niet alleen het type toch ?

Reply:
klopt, het woord "alleen" stond er niet tussen. Maar voor de duidelijkheid heb ik:
1. het woord "alleen" wel toegevoegd
2. fields gewijzigd naar "burgerservicenummer,verblijfplaats.postcode"